config/services/dataplane_v1beta1_openstackdataplaneservice_configurenetwork.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneService
+metadata:
+  labels:
+    app.kubernetes.io/name: openstackdataplaneservice
+    app.kubernetes.io/instance: openstackdataplaneservice-sample
+    app.kubernetes.io/part-of: dataplane-operator
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/created-by: dataplane-operator
+  name: configure-network
+spec:
+  label: dataplane-deployment-configure-network
+  role:
+
+    name: "Deploy EDPM Network"
+    hosts: "all"
+    strategy: "linear"
+    tasks:
+      - name: "Install edpm_bootstrap"
+        import_role:
+          name: "osp.edpm.edpm_bootstrap"
+          tasks_from: "bootstrap.yml"
+          tags:
+            - "edpm_bootstrap"
+      - name: "Grow volumes"
+        import_role:
+          name: "osp.edpm.edpm_growvols"
+          tasks_from: "main.yml"
+        tags: "edpm_growvols"
```

Comment:
Please change line 29 to the following so it takes line 29 and 30:
```
        tags:
          - "edpm_growvols"
```
Otherwise you'll get 
```
The OpenStackDataPlaneService "configure-network" is invalid: spec.role.tasks[1].tags: Invalid value: "string": spec.role.tasks[1].tags in body must be of type array: "string"
```

Reply:
Done.